src/components/SearchStatus/searchStatusStyles.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from "@mui/styles";
+import { Theme } from "@mui/material";
+
+export const useStyles = makeStyles(({ spacing, breakpoints, typography }: Theme) => ({
```

Comment:
This file will be moved to corresponding component when `RecentSearch` component is available.

Reply:
I made the component reusable so I can use it with RecentSearch and SearchResults.